internal/vds/vds_test.go

```diff
@@ -573,7 +572,7 @@ func TestOnly3DSupported(t *testing.T) {
 		handle.Close()
 	}
 
-	require.ErrorContainsf(t, err, "3 dimensions, got 4", "Incorrect error msg")
```

Comment:
why did you remove "got 4"?

Reply:
Because my vim macro was incorrect 😄 I'll fix it on squash